ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.scss

```diff
@@ -41,6 +41,14 @@
             }
         }
 
+        .argo-field,
+        .argo-textarea {
```

Comment:
Could you check if `.argo-textarea` is needed? The menu items are argo-fields

Reply:
hmm removed the `.argo-textarea` and the `color` now its consistent for all cases. also I have removed the include themify in the file for this PR to avoid merge conflicts. after the merge of #18424.